test/button_test.js

```diff
@@ -346,10 +355,13 @@ describe("Summary Ping Tests", function() {
       [ping => Object.hasOwnProperty.call(ping.payload.data.attributes, "summary")],
       pings);
     assert(foundPings.length > 0);
-    assert(JSON.parse(foundPings[0].payload.data.attributes.numberOfTimesURLBarCopied) === 3);
+    const urlBarCopies = JSON.parse(foundPings[0].payload.data.attributes
+      .numberOfTimesURLBarCopied);
+    assert(urlBarCopies === 3, `Expected 3 urlBarCopies, instead urlBarCopies = ${urlBarCopies}`);
   });
 
-  it("should report the correct number of share button clicks", async() => {
+  // NOTE This is simply testing built-in Firefox Telemetry pings.
+  /* it("should report the correct number of share button clicks", async() => {
```

Comment:
That seems reasonable. I think my original thought behind keeping these tests commented was that it might be useful to verify that the extension is looking at the correct Telemetry histogram. However, this would only really help in detection, but if the tests are inconsistent and commented then the detection would have to be done manually anyways.